src/Examples/UnitModels/Operations/heat_exchanger_NTU_testing.ipynb

```diff
@@ -0,0 +1,513 @@
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The initial DOF is 15\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Create an instance of the heat exchanger unit, attaching it to the flowsheet\n",
+    "#Specify that the property package to be used with the heater is the one we created earlier.\n",
+    "m.fs.heat_exchanger = HXNTU(\n",
+    "    hot_side_name=\"shell\",\n",
+    "    cold_side_name=\"tube\",\n",
+    "    shell={\"property_package\": m.fs.hotside_properties,\n",
+    "           \"has_pressure_change\": True},\n",
+    "    tube={\"property_package\": m.fs.coldside_properties,\n",
+    "          \"has_pressure_change\": True})\n",
+    "\n",
+    "# Call the degrees_of_freedom function, get initial DOF\n",
+    "DOF_initial = degrees_of_freedom(m)\n",
+    "print(\"The initial DOF is {0}\".format(DOF_initial))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "tags": [
+     "testing"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "assert DOF_initial == 15"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fixing input specifications\n",
+    "For this problem, we will fix the inlet conditions and run two different options for unit specifications:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The DOF is 3\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Hot fluid\n",
+    "m.fs.heat_exchanger.hot_side_inlet.flow_mol[0].fix(60.54879) # mol/s\n",
+    "m.fs.heat_exchanger.hot_side_inlet.temperature[0].fix(392.23) # K\n",
+    "m.fs.heat_exchanger.hot_side_inlet.pressure[0].fix(202650) # Pa\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"CO2\"].fix(0.0158) # dimensionless\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"H2O\"].fix(0.8747) # dimensionless\n",
+    "m.fs.heat_exchanger.hot_side_inlet.mole_frac_comp[0, \"MEA\"].fix(0.1095) # dimensionless\n",
+    "\n",
+    "# Cold fluid\n",
+    "m.fs.heat_exchanger.cold_side_inlet.flow_mol[0].fix(63.01910) # mol/s\n",
+    "m.fs.heat_exchanger.cold_side_inlet.temperature[0].fix(326.36) # K\n",
+    "m.fs.heat_exchanger.cold_side_inlet.pressure[0].fix(202650) # Pa\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"CO2\"].fix(0.0414) # dimensionless\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"H2O\"].fix(0.8509) # dimensionless\n",
+    "m.fs.heat_exchanger.cold_side_inlet.mole_frac_comp[0, \"MEA\"].fix(0.1077) # dimensionless\n",
+    "\n",
+    "DOF_initial = degrees_of_freedom(m)\n",
+    "print(\"The DOF is {0}\".format(DOF_initial))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Option 1: Fix HTC and dimensions of each domain\n",
```

Comment:
"HTC" needs to be expanded, as this is the first occurrence of it in this notebook.